lib/src/facts/windows/collection.cc

```diff
@@ -32,13 +33,26 @@ namespace facter { namespace facts {
 
     vector<string> collection::get_external_fact_directories()
     {
-        // Get the user data path
-        TCHAR szPath[MAX_PATH];
-        if (!SUCCEEDED(SHGetFolderPath(NULL, CSIDL_COMMON_APPDATA, NULL, 0, szPath))) {
-            LOG_DEBUG("error finding COMMON_APPDATA: %1%", system_error());
+        if (user::is_admin()) {
+            // Get the common data path
+            TCHAR szPath[MAX_PATH];
+            if (SUCCEEDED(SHGetFolderPath(NULL, CSIDL_COMMON_APPDATA, NULL, 0, szPath))) {
+                path p = path(szPath) / "PuppetLabs" / "facter" / "facts.d";
+                return {p.string()};
+            }
+
+            LOG_WARNING("error finding COMMON_APPDATA, external facts unavailable: %1%", system_error());
+        } else {
```

Comment:
Word. I'd say it's also surprising on Posix systems that non-admin users won't load global external facts, so it's something to revisit.